var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -70,6 +70,9 @@ class Gaudi(CMakePackage):
         when="@:38.1",
     )
 
+    # add a few missing includes (c++20?)
+    patch("includes.patch", when="@37:38")
```

Comment:
From your comments it sounds like this is in the master branch so we can close the range.
```suggestion
    patch("includes.patch", when="@37:38.3")
```

Reply:
I did not need this incremental change and 38.3 was the last release before 39, so I think this is optional for now.